numpy/fft/_helper.py

```diff
@@ -193,6 +200,13 @@ def rfftfreq(n, d=1.0):
         Window length.
     d : scalar, optional
         Sample spacing (inverse of the sampling rate). Defaults to 1.
+    device : str, optional
+        The device on which to place the created array. Default: ``None``.
+
+        .. note::
```

Comment:
There are several more `.. note::`'s that should be replaced by the more compact sentence

Reply:
Ah, right - missed two in `fft` module. Fixed!